go/src/koding/db/mongodb/modelhelper/group.go

```diff
@@ -21,6 +21,17 @@ func GetGroupById(id string) (*models.Group, error) {
 	return group, Mongo.Run(GroupsCollectionName, query)
 }
 
+// GetGroupsByIds returns groups by their given IDs
+func GetGroupsByIds(ids ...bson.ObjectId) ([]*models.Group, error) {
+	groups := make([]*models.Group, 0)
```

Comment:
nitpick: can be just `var groups []*models.Group`

https://github.com/golang/go/wiki/CodeReviewComments#declaring-empty-slices